networkx/algorithms/traversal/depth_first_search.py

```diff
@@ -15,7 +15,7 @@
 
 
 @nx._dispatchable
-def dfs_edges(G, source=None, depth_limit=None):
+def dfs_edges(G, source=None, depth_limit=None, sort_neighbors=None):
```

Comment:
Since these kwargs are new in the dfs fns, let's make them keyword only!

```suggestion
def dfs_edges(G, source=None, depth_limit=None, *, sort_neighbors=None):
```